src/test/java/service/FruitServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package service;
+
+import static org.junit.Assert.assertEquals;
+
+import db.Storage;
+import model.Fruit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static final String SYMBOL_FOR_SEPARATING = ",";
+    private static Storage storage;
+
+    @BeforeClass
+    public static void operationBeforeTest() {
+        storage = new Storage();
+    }
```

Comment:
We don't need to create an instance of Storage class. We can reference our map using Storage.storage because it is static.